recipe_scrapers/streetkitchen.py

```diff
@@ -11,7 +12,15 @@ def title(self):
         return self.soup.find("h1", {"class": "entry-title"}).get_text()
 
     def total_time(self):
-        return None
+        items = self.soup.select(".the-content-div li")
+        total_time = 0
+
+        for item in items:
+            text = item.get_text()
+            if "perc" in text:
+                total_time += int(text.split(" ")[-2])
+
+        return total_time if total_time != 0 else None
```

Comment:
We have support for matching `perc` as minutes in our `get_minutes` utility function - could that work here?

```suggestion
        total_time = 0
        for item in items:
            total_time += get_minutes(item.text) or 0
        return total_time or None
```

(note: needs an `import` too if so!)

If it's not usable / broken for some other pages, that would be good to know (more problems, more fixes).